package.json

```diff
@@ -54,7 +54,7 @@
     "karma-chrome-launcher": "^2.2.0",
     "karma-firefox-launcher": "^1.0.1",
     "karma-mocha": "^1.3.0",
-    "karma-mocha-own-reporter": "^1.1.2",
+    "karma-mocha-own-reporter": "git+https://github.com/dryajov/karma-mocha-own-reporter",
```

Comment:
can you change it, so it references a commit, to avoid any chance of accidental breakage?